docs/cookbook.rst

```diff
@@ -221,8 +233,11 @@ Would generate this log::
                         ... hoping it's equal to True.
                             => <True>
 
-But what if perhaps, we didn't need to know all the steps being taken at in  ``PerformChatty`` unless they were to fail?
-Wrapping ``PerformA`` in ``Silently``::
+But what if perhaps, we didn't need to know all the steps being taken at in
+``PerformChatty`` unless they were to fail?
+Wrapping ``PerformA`` in ``Silently``
```

Comment:
I think we need to rework this a bit.

```rst
That is a lot of information.
If we only want to know
the name of the outside task—
unless a failure occurs—
we can use :class:`~screenpy.actions.Silently`
to wrap ``PerformA`` inside of ``PerformChatty``:

.. code-block::python
```

I also applied the [semantic linebreak](https://sembr.org/) style that's used elsewhere in our docs up there.

Reply:
I fixed this one and one other in that area, but I think a separate PR should be created to address the rest of them.  I didn't understand why they looked that way when I wrote those in a previous PR.